taurus/util/impl.py

```diff
@@ -181,7 +189,7 @@ def recursive_foreach(obj, func, apply_first=False, seen=None):
     return
 
 
-def recursive_flatmap(obj, func, seen=None):
+def recursive_flatmap(obj, func, seen=None, chronological=False):
```

Comment:
Suggestion template:
`@param chronological causes func to be called in reverse chronological order, rather than the default _____ ordering`